manifests/g/GIMP/GIMP/2.10.24/GIMP.GIMP.installer.yaml

```
@@ -1,17 +1,151 @@
-# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: GIMP.GIMP
 PackageVersion: 2.10.24
+Platform:
+- Windows.Desktop
 MinimumOSVersion: 10.0.0.0
 InstallerType: inno
-InstallerSwitches:
-  Silent: /VERYSILENT
-  SilentWithProgress: /SILENT
-  Custom: /NORESTART /ALLUSERS
+UpgradeBehavior: install
+FileExtensions:
+- 3fr
+- alpha
+- als
+- apm
+- ari
+- arw
+- avif
+- bay
+- bitmap
+- bmp
+- bw
+- cap
+- cel
+- cin
+- cine
+- cr2
+- cr3
+- crw
+- cs1
+- data
+- dc2
+- dcm
+- dcr
+- dds
+- desktop
+- dicom
+- dng
+- eps
+- erf
+- exr
+- fff
+- fit
+- fits
+- flc
+- fli
+- g3
+- gbp
+- gbr
+- gif
+- gih
+- hdr
+- heic
+- heif
+- hgt
+- ia
+- icb
+- ico
+- icon
+- iiq
+- im1
+- im24
+- im32
+- im8
+- j2c
+- j2k
+- jp2
+- jpc
+- jpe
+- jpeg
+- jpg
+- k25
+- kc2
+- kdc
+- mask
+- matte
+- mdc
+- mef
+- mos
+- mrw
+- nef
+- nrw
+- ora
+- orf
+- pat
+- pbm
+- pcc
+- pcx
+- pdf
+- pef
+- pfm
+- pgm
+- pix
+- png
+- pnm
+- ppm
+- ps
+- psd
+- psp
+- pspimage
+- pxn
+- qtk
+- ra
+- raf
+- ras
+- raw
+- rdc
+- rgb
+- rgba
+- rw2
+- rwl
+- sgi
+- sr2
+- srf
+- srw
+- sti
+- sun
+- svg
+- tga
+- tif
+- tiff
+- tub
+- vda
+- vst
+- webp
+- wmf
+- x3f
+- xbm
+- xcf
+- xcf.bz2
+- xcf.gz
+- xcfbz2
+- xcfgz
+- xcfxz
+- xpm
+- xwd
 Installers:
 - Architecture: x64
-  InstallerUrl: https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.24-setup-1.exe
-  InstallerSha256: 84A542D717217F5FB996A18093C4424136978EBBF3815DB2EE3C5DAA8DCAFC0B
+  Scope: machine
+  InstallerUrl: https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.24-setup-3.exe
+  InstallerSha256: 5E9EABE5739523A9FC347B4614D919418F3335E7AAB082A65F71705421E85E04
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- Architecture: x64
+  Scope: user
+  InstallerUrl: https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.24-setup-3.exe
+  InstallerSha256: 5E9EABE5739523A9FC347B4614D919418F3335E7AAB082A65F71705421E85E04
+  InstallerSwitches:
+    Custom: /CURRENTUSER
```

I would add the x86 entries
```suggestion
  Scope: machine
  InstallerUrl: https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.24-setup-3.exe
  InstallerSha256: 5E9EABE5739523A9FC347B4614D919418F3335E7AAB082A65F71705421E85E04
  InstallerSwitches:
    Custom: /ALLUSERS
- Architecture: x64
  Scope: user
  InstallerUrl: https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.24-setup-3.exe
  InstallerSha256: 5E9EABE5739523A9FC347B4614D919418F3335E7AAB082A65F71705421E85E04
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  Scope: machine
  InstallerUrl: https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.24-setup-3.exe
  InstallerSha256: 5E9EABE5739523A9FC347B4614D919418F3335E7AAB082A65F71705421E85E04
  InstallerSwitches:
    Custom: /ALLUSERS
- Architecture: x86
  Scope: user
  InstallerUrl: https://download.gimp.org/pub/gimp/v2.10/windows/gimp-2.10.24-setup-3.exe
  InstallerSha256: 5E9EABE5739523A9FC347B4614D919418F3335E7AAB082A65F71705421E85E04
  InstallerSwitches:
    Custom: /CURRENTUSER
```